internal/pkg/vm/internal/vmcontext/invocation_context.go

```diff
@@ -88,7 +89,7 @@ func (ctx *invocationContext) invoke() interface{} {
 
 	// 4. if we are just sending funds, there is nothing else to do.
 	if ctx.msg.method == builtin.MethodSend {
-		return nil
+		return &adt_spec.EmptyValue{}
```

Comment:
the return here is encoded to bytes and placed in the message receipt here: https://github.com/filecoin-project/go-filecoin/blob/d12a39ac0927124eebd34194eda304d94b52f0aa/internal/pkg/vm/internal/vmcontext/vmcontext.go#L427

`nil` is not an encodable value (this a bug with encode?), returning something that encodes to an empty return value.